CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v1.5.0 / 2018-01-10
+
+After a testing period of 30 days, there were no additional bugs found or features introduced. Due to no bugs being reported over an in total 41 days, we feel no more pre-releases are necessary for a stable release.
+
+This release's focus was a large architectural change in order to improve performance and resource usage of kube-state-metrics drastically. Special thanks to @mxinden for his hard work on this! See the changelog of the pre-releases for more detailed information and related pull requests.
+
+An additional change has been requested to be listed in the release notes:
+
+* [CHANGE] Due to removal of the surrounding mechanism the `ksm_resources_per_scrape` metric no longer exists.
```

Comment:
Small nit-pick: Both `ksm_resources_per_scrape` and `ksm_scrape_error_total` metrics actually.

Reply:
You're absolutely right! Adapted.